### ISSUE TITLE
The video's look weird

### ISSUE THREAD

**Issue by fat2slow** [YePpHa/YouTubeCenter#1697]
![screenshot 2015-04-28 22 18 16](https://cloud.githubusercontent.com/assets/10214549/7384260/0beb5f26-edf5-11e4-8060-7a2fb97e0abc.png)
all the stuff keeps pushing up into the Video's region

**Comment by BiZZKeryear** [comment]
I can replicate this, seems 2.1.7 is finally total broken.

**Comment by jasonycw** [comment]
Seems Youtube is updated recently, everything goes wrong for me too just a few minutes ago.

**Comment by aztectornado** [comment]
Yeah, youtube updated and changed a lot about the way the UI is figured out. It's a mess.

**Comment by endymion0000** [comment]
Yeah I'm seeing a shifted player window when using the 720p player size.

**Comment by SINE** [comment]
Related
#1695
#1696

**Comment by KamranMackey** [comment]
Yeah, i'm experiencing the same issue in the dev version as well. @YePpHa could you please fix this? :smiley:

**Comment by Corruptinglyneedful** [comment]
I am also having this issue.

**Comment by SuperSajuuk** [comment]
Yep, apparently caused by a style update. But the update has finally broken the stable build and it should really be taken down and users should be asked to use the developer build at all times :/
